encoding/thrift/thriftrw-plugin-yarpc/client.go

```diff
@@ -107,7 +108,12 @@ func (c client) <.Name>(
 	args := <$prefix>Helper.Args(<range .Arguments>_<.Name>, <end>)
 
 	var body <$wire>.Value
+	<if $sanitize>
+	<$tchannel := import "github.com/uber/tchannel-go">
+	body, err = c.c.Call(<$tchannel>.WithoutHeaders(ctx), args, opts...)
+	<else>
 	body, err = c.c.Call(ctx, args, opts...)
+	<end>
```

Comment:
Nit: Instead, can we just do this above the Call?

```
<if $sanitize>ctx = <import "github.com/uber/tchannel-go">.WithoutHeaders(ctx)<end>
```

No other code has to change that way.